backend/src/main/java/de/neuefische/finalproject/ohboy/controller/MonsterController.java

```diff
@@ -47,4 +48,12 @@ public Monster update(@RequestBody UpdateMonsterDto updatedMonster, @PathVariabl
         return monsterService.update(updatedMonster);
     }
 
+    @DeleteMapping("{monsterId}")
+    public void remove(@RequestBody RemoveMonsterDto removedMonster, @PathVariable String monsterId) {
```

Comment:
consider using another name instead of "removedMonster" since the monster is not yet removed at this stage